src/layouts/Extensions.vue

```diff
@@ -66,8 +130,46 @@ export default {
 	data() {
 		return {
 			extensions: [],
+			filters: {
+				search: "",
+				lang: [],
```

Comment:
Currently lang will never be an array
```suggestion
				lang: "",
```